vrk-kpa/suomidigi
# SUOPA-645 Profile

This PR includes SUOPA-663 (Theming of the profile page).

To test: 
- `make fresh`
- Clear the caches
- With incognito browser head to https://suomidigi.docker.sh. 
   - Check that there is login link (Kirjaudu sisään) next to language selection and search input field
   - Click it and log in as a normal user who has no roles or max content producer role (no admin nor editor in chief)
- After logging in, check that the login link has changed to small menu-like tool like below. 
![Suomidigi 2019-10-28 15-16-45](https://user-images.githubusercontent.com/1712902/67683734-90bb8900-f99a-11e9-9122-06bc0e1f0f25.jpg)
- Click on Edit profile (muokkaa profiilia) and check that the form looks good and is easy to understand. 
- Fill in all of the form fields and save the profile. Click on View profile (näytä profiili) and check that the profile looks somewhat like this 
![vrk-suomidigi-oct-2019-uusi 2019-10-29 16-01-35](https://user-images.githubusercontent.com/1712902/67774138-81edd880-fa65-11e9-882b-ac9885a55535.jpg)
